lib/qs/payload_handler.rb

```diff
@@ -70,12 +70,12 @@ def raise_if_debugging!(exception)
     end
 
     def log_received
-      log_verbose "===== Running job ====="
+      log_verbose "===== Received message ====="
     end
 
-    def log_job(job)
-      log_verbose "  Job:     #{job.name.inspect}"
-      log_verbose "  Params:  #{job.params.inspect}"
+    def log_message(message)
+      log_verbose "  Name:    #{message.route_id.inspect}"
+      log_verbose "  Params:  #{message.params.inspect}"
```

Comment:
@jcredding didn't you want to do some specific logging depending on whether it was a job or event message?